browser/ui/views/tabs/dragging/tab_drag_controller_browsertest.cc

```diff
@@ -1,81 +0,0 @@
-/* Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
Deleted as this test is for checking DnD crash failre when BraveSplitView is enabled.
Original PR: https://github.com/brave/brave-core/pull/25160